app/assets/javascripts/miq_novnc.js

```diff
@@ -1,6 +1,8 @@
 "use strict";
 var rfb;
 
+window.WEB_SOCKET_SWF_LOCATION = '/assets/noVNC/web-socket-js/WebSocketMain.swf';
```

Comment:
@martinpovolny Should we follow the pattern in this [discussion](https://groups.google.com/forum/#!topic/socket_io/UAc6TRsqnys)?